lib/processors/minifierWorker.js

```diff
@@ -40,9 +40,9 @@ export default async function execMinification({
 	sourceMapOptions
 }) {
 	try {
-		return await minify({
+		const result = await minify({
 			// Use debug-name since this will be referenced in the source map "sources"
-			[dbgFilename]: code
+			[dbgFilename]: Buffer.from(code).toString()
```

Comment:
I got some wird results when calling `Buffer.toString()` without encoding in https://github.com/SAP/ui5-builder/pull/914 when (de-)serializing `@ui5/fs/Resources`.
Maybe, it's a good point to add an encoding. I guess, if those are just `js` files, we can asume `utf8`

Reply:
That's strange as we're using it like that ever since in ui5-fs: https://github.com/SAP/ui5-fs/blob/f378330dc1da077b6f945d59c41592b776a7001e/lib/Resource.js#L175

Reply:
With binary files such as images, it doesn't work for obvious reasons, though.

Reply:
Sorry, I was refering to this one: https://github.com/SAP/ui5-fs/blob/f378330dc1da077b6f945d59c41592b776a7001e/lib/Resource.js#L185

Reply:
If the first argument is a string, the encoding defaults to utf-8. At least that's how the function is documented in Node.js